services/director/src/simcore_service_director/cache_request_decorator.py

```diff
@@ -6,25 +6,25 @@
 
 from simcore_service_director import config
 
-def cache_requests(http_request):
+
+def cache_requests(http_request: callable):
     @wraps(http_request)
     async def wrapped(app: web.Application, url: URL, method: str ="GET") -> Tuple[Dict, Dict]:
-        if config.REGISTRY_CACHING:
+        if config.DIRECTOR_REGISTRY_CACHING and method == "GET":
```

Comment:
MINOR: more readable if defined as a variable at the top as

```python
is_cache_enabled = config.DIRECTOR_REGISTRY_CACHING and method=="GET"

if is_cache_enabled:
  ...

if is_cache_enabled:
...

``

Reply:
done